### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by pihart**

### CODE REVIEW

File: src/main.scss [hunk 1]
```diff
@@ -1,17 +1,76 @@
...
-@media all and (max-width:600px) {body {font-size: 15px}}
-@media all and (max-width:560px) {body {font-size: 14px}}
-@media all and (max-width:520px) {body {font-size: 13px}}
-@media all and (max-width:400px) {body {font-size: 12px}}
-@media all and (max-width:360px) {body {font-size: 11px}}
-@media all and (max-width:330px) {body {font-size: 10px}}
-@media all and (max-width:300px) {body {font-size: 9px}}
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+@media all and (min-width: 760px) {
```
Comment (cjquines) [inline src/main.scss:3]: i am thoroughly disgusted by this there's probably a nice scss way to do this

Comment (cjquines) [reply src/main.scss:3]: probably https://sass-lang.com/documentation/at-rules/control/each#with-maps

yeah i shouldve done this in the first place but this is originally from raw css

**Review by cjquines — CHANGES_REQUESTED** [review]

File: src/main.scss [hunk 2]
```diff
@@ -21,7 +80,9 @@ body {
   padding: 0;
 }
 
-html, body, .container {
```
Comment (cjquines) [inline src/main.scss:24]: i dont like this rule :( convince me?

Comment (pihart) [reply src/main.scss:24]: Hard to discern, consistency with other scss files.

(I used to hate this rule too)

File: src/main.scss (hunk 1 above)
Comment (cjquines) [reply src/main.scss:3]: i'll make this

Comment (pihart) [reply src/main.scss:3]: Oh I didn't even realize it was counting down by a contant difference. Let me try to fix

Comment (cjquines) [reply src/main.scss:16]: uh too late. does it look good

**Comment by pihart** [comment]
Wait the recent commit isn't the right way

**Comment by cjquines** [comment]
it's not a constant difference

**Review by cjquines — APPROVED** [review]

**Pull request merged**